# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1458613222 14400
#      Mon Mar 21 22:20:22 2016 -0400
# Branch stable
# Node ID f744aa689d94cac45138bc99adf3d0b1fc71ba9e
# Parent  5162cdfb56d8e31ad00dc260a26d1c92c69c2acf
Added tag release-4-0-1 for changeset 5162cdfb56d8

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -93,8 +93,9 @@ a66548dc07b0fc9b7d37a4be065aa15e8243c96a
 52d2bbf49c922440621b42b1c1679f0e55462485 rc-4-0-0-1
 eba80000fa0dad32ba0f1cd767dd826d1ce1aba6 rc-4-0-0-2
 065f933ef08318e40b81f7fe75236e6175088117 rc-4-0-0-3
 42bb3a776c9fcc008669f382d2409297c4a901b3 rc-4-0-0-4
 20b17dda0e0c838646dfdccec153ec8cdf6d3c37 rc-4-0-1-1
 0cfcc178432bfc7cfdef5dca96e74ec5884ba752 rc-4-0-1-2
 a58d5e6df908199c4aea83f78077b81289d2a0ad rc-4-0-1-3
 2b66f30a0de7a09801c2a3566021cde7e194d3d3 rc-4-0-1-4
+5162cdfb56d8e31ad00dc260a26d1c92c69c2acf release-4-0-1
